pacemaker.spec.in

```diff
@@ -355,6 +355,10 @@ install -m 644 mcp/pacemaker.combined.upstart ${RPM_BUILD_ROOT}%{_sysconfdir}/in
 install -m 644 tools/crm_mon.upstart ${RPM_BUILD_ROOT}%{_sysconfdir}/init/crm_mon.conf
 %endif
 
+%if %{defined _unitdir}
+mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/lib/rpm-state/%{name}
+%endif
```

Comment:
Perhaps we should just own the directory with %dir, and %ghost only the file? Shared stuff is typically owned by the -cli package since everything depends on that.